### ISSUE TITLE
Project title

### ISSUE THREAD

**Issue by 7yl4r** [ioos/ioos-code-sprint#9]
## Project Description:

Open source biodiversity indicator development from open data.

## Expected Outcomes: 

This project aims to advance the state of biodiversity indicators generated from open data.
I intend to document efforts using jupyter notebooks.
Accessibility of Jupyter notebooks will be enhanced using mybinder.org.
My preferred language for this analysis is python but R+julia will also be available.
Visualization & statistics from data will be pursued.
Analyses will specifically focus on one of the three: 1) spatial mapping of biodiversity, 2) biodiversity time-series, 3) species-specific comparisons.

## Skills required:

Anybody familiar with python/R/julia, OBIS/GBIF/DwC, or biodiversity indices are welcome.

## Difficulty:

Moderate

## Topic Lead(s):

Tylar Murray @7yl4r

## Relevant links:

* PR #8
* https://github.com/marinebon/obis2index
* https://github.com/USF-IMARS/obis_notebooks
* https://github.com/marinebon/datatransformers

**Comment by MathewBiddle** [comment]
project page started https://ioos.github.io/ioos-code-sprint/projects/04-biodiversity-indicator-development/

**Comment by 7yl4r** [comment]
One apprehension I have that the mybinder.org service at the core of my plan can be slow at times. We may want to spin up a more performant jupyterhub server to use for this. Something else worth considering is trying to leverage Axiom's "Research Workspace" - for that we need to discuss with someone at Axiom.

**Comment by MathewBiddle** [comment]
What about [google colab](https://colab.research.google.com/)? Put a bunch of `try:` `except` statements in for installing packages w/ `pip`.

There's also a way to use [R in google colab](https://towardsdatascience.com/how-to-use-r-in-google-colab-b6e02d736497), which could be interesting to explore.

Or, for the sprint you build something that is ran locally. Then, after the sprint, look at ways to integrate with these other services?

**Comment by MathewBiddle** [comment]
@7yl4r are you okay giving this sprint pitch on Monday? We are looking for a brief overview of what the topic will be, what slack channel, and what breakout room to join the effort (along with anything else that might be relevant).

**Comment by 7yl4r** [comment]
Yes. I plan to just show the topic page. Should we add the slack channel and breakout room info to that page?

**Comment by MathewBiddle** [comment]
I'll present the page if you're up to speaking about it. I've added the slack channel and breakout room to the webpage, but feel free to update as appropriate.

**Comment by yvanlebras** [comment]
amazing topic! Sorry to discover it now the code sprint is finished.... FYI we are working on such topic (giving access to open source codes / workflows to compute biodiversity indicators) through the [Galaxy-E](https://galaxyproject.org/use/ecology/) initiative. Briefly, we are focusing on producing biodiveristy indicators through open source workflows (using R / Python / others languages scripts)  working notably on 1/ generalization so a workflow can be used widely (spatially and/or taxonomicaly (if frenglish ;) ) notably) 2/ atomization (so each step of a workflow can be reused in another workflow / indepentantly 3/ accessibility-transparency-reproductibility using the Galaxy platform allowing people without knowledge in programming skills to execute these tools/workflows + facilitating sharing the workflow and resulting execution + provenance tracking. If you are interested, don't hesitate to come back to me!

Bonus => you can deploy RStudio or Jupyter notebooks directly from Galaxy if you want to interact with codes on Galaxy datasets!

**Comment by 7yl4r** [comment]
closing since we sprint is finished.
